docs/tutorial/deployments.md

```diff
@@ -146,29 +135,17 @@ if __name__ == "__main__":
 
 A few observations:
 
-- the `flow.to_deployment` interface exposes the *exact same* options as `flow.serve`; this method produces a deployment object
-- the deployments are only registered with the API once `serve(...)` is called
-- when serving multiple deployments, the only requirement is that they share a Python environment; they can be executed and scheduled independently of each other
-
-Spend some time experimenting with this setup.
-A few potential next steps for exploration include:
+- The `flow.to_deployment` interface exposes the *exact same* options as `flow.serve`; this method produces a deployment object.
+- Register deployments with the API after you call `serve(...)`.
```

Comment:
```suggestion
- The script only registers the deployment with the API when `serve(...)` is called.
```

Reply:
As written, it sounds like the user needs to take an action - which isn't the case.

Reply:
fwiw, my 2 cents here

> Calling `.serve()` registers the flow as a deployment - you can now trigger it on a schedule or on events.